src/libraries/Microsoft.Extensions.Primitives/ref/Microsoft.Extensions.Primitives.cs

```diff
@@ -58,8 +59,8 @@ public partial interface IChangeToken
         public bool Equals(Microsoft.Extensions.Primitives.StringSegment other) { throw null; }
         public static bool Equals(Microsoft.Extensions.Primitives.StringSegment a, Microsoft.Extensions.Primitives.StringSegment b, System.StringComparison comparisonType) { throw null; }
         public bool Equals(Microsoft.Extensions.Primitives.StringSegment other, System.StringComparison comparisonType) { throw null; }
-        public override bool Equals(object obj) { throw null; }
-        public bool Equals(string text) { throw null; }
+        public override bool Equals(object? obj) { throw null; }
```

Comment:
Thanks for the tip! This might mon actually be the case in this case. Since `Equals` will return `true` if `StringValue` contains `null` or empty string.

Reply:
This is StringSegment, not StringValue.  StringSegment.Equals(object) is defined as:
```C#
       public override bool Equals(object obj)
        {
            return obj is StringSegment segment && Equals(segment);
        }
```

Reply:
I think the only overload where this is applicable is this one. I'll add it there.

![image](https://user-images.githubusercontent.com/6609929/129457375-4b10db7a-e315-4975-8ab5-98deafabb7e4.png)

Reply:
Sorry, didn't notice the type you were commenting on

Reply:
Fixed